java/gradle.java/arch.xml

```diff
@@ -110,6 +110,18 @@
          in action's context Lookup. See <a href="@TOP@/org/netbeans/modules/gradle/java/api/ProjectActions.html#TOKEN_JAVAEXEC_ARGS">ProjectActions.TOKEN_JAVAEXEC_ARGS</a>
          for more details.
      </api>
+     <api category="stable" group="property" name="javaExec.runWorkingDir" type="export">
```

Comment:
Actually this is `type="import"` as you are importing/using the Gradle API here. Thanks for documenting the contract anyway!

Reply:
Well, there's `export` for other properties as well. IMHO the idea is, that these properties are created from `ExplicitProcessParameters`. We are essentially providing property values that are used in `NetBeansRunSinglePlugin.groovy` then.